### PR TITLE
Automatic versions update

### PR DESCRIPTION
**Pull request by pavelkornev**

### CODE REVIEW

File: bin/versions/fetchVersions.mjs [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import fetch from "node-fetch";
+import { writeFileSync } from "fs";
+import path from "path";
+
+const __dirname = path.dirname(new URL(import.meta.url).pathname);
+const FILE_PATH = path.resolve(__dirname, "../data/pages/releases/versions.json");
+
+(async () => {
+  try {
+    const response = await fetch("https://openui5versiontracker.cfapps.eu10.hana.ondemand.com/OpenUI5ReleasesInfo");
```
Comment (evgenysegeda) [inline bin/versions/fetchVersions.mjs:10]: is it possible to move to the variable?

Comment (pavelkornev) [reply bin/versions/fetchVersions.mjs:10]: move what?

Comment (evgenysegeda) [reply bin/versions/fetchVersions.mjs:10]: I meant the url. Is it possible as in the script?

Comment (pavelkornev) [reply bin/versions/fetchVersions.mjs]: done

**Pull request merged**